bigtable/tests/unit/test_row_filters.py

```diff
@@ -295,10 +295,12 @@ def _to_pb_helper(self, start_micros=None, end_micros=None):
         if start_micros is not None:
             start = _EPOCH + datetime.timedelta(microseconds=start_micros)
             pb_kwargs['start_timestamp_micros'] = start_micros
+            self.assertEqual(start_micros % 1000, 0)
         end = None
         if end_micros is not None:
             end = _EPOCH + datetime.timedelta(microseconds=end_micros)
             pb_kwargs['end_timestamp_micros'] = end_micros
```

Comment:
Could you test that the rounding up behavior is as you expect? Use a specific value for the expected end_micros would make the test more readable.